examples/sensor-graph/src/components/SensorPlotGraph.jsx

```diff
@@ -0,0 +1,188 @@
+      window.confirm("Device ID Invalid")
+    }
+  }
+
+
+  setDeviceDataById = (id, params = {}) => {
+    getDeviceDataById(id, params)
+      .then(res => {
+          this.setState({data: res.data, loading: false, fetched: true, ValueInterfaces: res.interfaces[res.valueIndex]});
+          getInterfaceById(id, res.interfaces[res.availableIndex])
+            .then(response => {
+              this.setState({availableSensors: response});
+            });
+          getInterfaceById(id, res.interfaces[res.valueIndex])
+            .then(response => {
+              this.setState({sensorValues: response});
+            })
+        }
+      ).catch((err) => {
+      this.handleError(err)
+    })
+  };
+
+
+  setDeviceDataByAlias = (alias, params = {}) => {
+    getDeviceDataByAlias(alias, params)
+      .then(res => {
+          this.setState({data: res.data, loading: false, fetched: true, ValueInterfaces: res.interfaces[res.valueIndex]});
+          getInterfaceByAlias(alias, res.interfaces[res.availableIndex])
+            .then(response => {
+              this.setState({availableSensors: response});
+            });
+          getInterfaceByAlias(alias, res.interfaces[res.valueIndex])
+            .then(response => {
+              this.setState({sensorValues: response});
+            })
+        }
+      ).catch((err) => {
+      this.handleError(err)
+    })
+  };
+
+
+  render() {
+    const {
+      visible,
+      sensorValues,
+      loading,
+      deviceID,
+      availableSensors,
+      ValueInterfaces,
+    } = this.state;
+    return (
+      <Container className="px-0 py-4" fluid>
+        <Container>
+          <Row>
+            <Col xs={12} className="p-0">
+              <h5
+                className="sensor-main-div text-center text-uppercase font-weight-bold text-white bg-sensor-theme mb-5 px-0 py-3">Sensor
+                Plot Example</h5>
+              <Col xs={12} className="sensor-id-search-div px-5 py-5">
+                <Row className="main-row col-sm-7 align-items-center mx-auto">
+                  <Col sm={2} xs={12} className={"p-0"}>
+                    <span className="font-weight-normal">Device ID:</span>
+                  </Col>
+                  <Col sm={10} xs={12}>
+                    <InputGroup>
+                      <FormControl
+                        placeholder="Enter Device ID Here"
+                        aria-label="Enter Device ID Here"
+                        aria-describedby="basic"
+                        className="bg-white font-weight-normal rounded"
+                        name="deviceID"
+                        onChange={this.handleChange}
+                      />
+                      <InputGroup.Append className="ml-3">
+                        <Button onClick={this.handleSubmit} disabled={loading}
+                                className="bg-sensor-theme border-success text-uppercase font-weight-normal px-4 text-decoration-none rounded">
```

Comment:
why `border-success` on this button?
"warning" "success" variants shouldn't be used just for the color. They have specific meanings and usage scenarios.
If you just want a specific color, use color classes